doc/classes/EditorSettings.xml

```diff
@@ -734,6 +841,12 @@
 			Depending on the platform and used renderer, the engine will fall back to [b]Enabled[/b] if the desired mode is not supported.
 			[b]Note:[/b] V-Sync modes other than [b]Enabled[/b] are only supported in the Forward+ and Mobile rendering methods, not Compatibility.
 		</member>
+		<member name="interface/editors/derive_script_globals_by_name" type="bool" setter="" getter="">
+			If [code]true[/code], when extending a custom script and it has a global name, the Inherits field of script creation dialog will be filled with the global name instead of script path.
```

Comment:
```suggestion
			If [code]true[/code], when extending a script, the global class name of the script is inserted in the "Create New Script" dialog, if it exists. Otherwise, If [code]false[/code], the script's file path is always inserted.
```

Reply:
Note that the dialog has different name when used from FileSystem dock, from Scene dock or from the Inspector.

Reply:
In that case could it not be...?
```suggestion
			If [code]true[/code], when extending a script, the global class name of the script is inserted in the script creation dialog, if it exists. Otherwise, If [code]false[/code], the script's file path is always inserted.
```